appveyor.yml

```diff
@@ -57,4 +62,14 @@ deploy:
     secure: QqxdbpN5rJhuQw42Iq4W6tdP1Lu4aw6y88QnHYC3tSQ9rzNuEI/q+qH4HGJ5WDpj
   on:
     branch: master                 # release from master branch only
-    appveyor_repo_tag: true        # deploy on tag push only
\ No newline at end of file
+    appveyor_repo_tag: true        # deploy on tag push only
+
+deploy_script:
```

Comment:
I haven't tested if both the `deploy:` and `deploy_script:` entries can work together in one appveyor.yml.